python/graphstorm/gconstruct/transform.py

```diff
@@ -272,11 +415,45 @@ def parse_feat_ops(confs):
                                                 int(conf['max_seq_length'])),
                                       conf['bert_model'],
                                       infer_batch_size=infer_batch_size)
+            elif conf['name'] == 'float_max_min':
```

Comment:
how about call it `max_min_norm`? you are actually doing normalization.

Reply:
max_min_norm is better